drivers/volume/portworx/portworx.go

```diff
@@ -568,6 +568,10 @@ func (d *portworx) StopDriver(nodes []node.Node, force bool) error {
 				return err
 			}
 		} else {
+			err = d.schedOps.DisableOnNode(n)
```

Comment:
In the start driver impl, you need to ensure the label is reset. Just call the start sched-ops API and that will add px/service=start. It will be harmless.

Reply:
Fixed.